web/stories/Molecules/Cards/OpportunityCards.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { OpportunityCard } from "@/components/dashboard/OpportunityCard";
+import { Opportunity } from "@/lib/types/types";
+import { generateOpportunity } from "@/test/factories";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+export default {
+  title: "Molecules/Cards/OpportunityCards",
+  component: OpportunityCard,
+  decorators: [(Story) => <div className="width-mobile-lg">{Story()}</div>],
+
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/vAa8neaM0JJSmldck5vlBC/BFS-Design-System?type=design&node-id=3266-13862&mode=design&t=3BT1qFE9mGxbn6Ko-0",
+    },
+  },
+} as ComponentMeta<typeof OpportunityCard>;
+
+type Props = {
+  opportunity: Opportunity;
+  urlPath: "funding" | "certification";
+  isLast?: boolean;
+};
+
+const Template: ComponentStory<typeof OpportunityCard> = ({ ...args }) => <OpportunityCard {...args} />;
+
+export const FundingCard = Template.bind({});
+const fundingOpportunity = generateOpportunity({ name: "Test Name for Card" });
```

Comment:
[pebble]: could be inlined? but truely fine either way

Reply:
I like this better declared as a variable.

Reply:
done